falcon/request.py

```diff
@@ -537,10 +539,17 @@ def app(self):
         return self.env.get('SCRIPT_NAME', '')
 
     @property
-    def protocol(self):
+    def scheme(self):
         return self.env['wsgi.url_scheme']
 
     @property
+    def protocol(self):
```

Comment:
Didn't know that earlier, nice one. updated [https://github.com/falconry/falcon/pull/917/commits/d3ce2c40eb2276183e845d9c384b28bfcf694586#diff-8608936f256c7b35254ee64f862001b9R545](https://github.com/falconry/falcon/pull/917/commits/d3ce2c40eb2276183e845d9c384b28bfcf694586#diff-8608936f256c7b35254ee64f862001b9R545)